coalib/results/result_actions/IgnoreResultAction.py

```diff
@@ -0,0 +1,35 @@
+from coalib.results.result_actions.ShowPatchAction import ShowPatchAction
+from coalib.results.result_actions.ResultAction import ResultAction
+import os.path
+
+
+class IgnoreResultAction(ResultAction):
+
+    SUCCESS_MESSAGE = "Result ignored successfully."
+
+    is_applicable = staticmethod(ShowPatchAction.is_applicable)
+
+    def apply(self, result, original_file_dict, file_diff_dict):
+        """
+        Ignore result
+        """
+        comment_delimiter = ""
+        for filename in result.diffs:
+            file_extension = os.path.splitext(os.path.basename(filename))[1]
+            file_extension = file_extension[1:]
+            if file_extension == "py":
+                file_extension = "python3"
```

Comment:
you shouldn't do this line ending based, rather add a `language: str` parameter to the signature